patch/llpcPatchEntryPointMutate.cpp

```diff
@@ -1051,6 +1051,7 @@ FunctionType* PatchEntryPointMutate::GenerateEntryPointType(
                     *pInRegMask |= 1ull << argTys.size();
                     entryArgIdxs.vs.streamOutData.writeIndex = argTys.size();
                     argTys.push_back(m_pContext->Int32Ty()); // Stream-out write Index
+                    entryArgIdxs.tes.streamOutData.writeIndex = argTys.size();
```

Comment:
This should be in the same place as for vs. In fact, why are any of the **entryArgIdx.*vs*.streamOutData written here, in the ShaderStageTessEval case?

Generally speaking, this whole code should probably refactored: there should be a function `SetupStreamOutArgs` which takes a pointer to the StreamOutData structure as well as additional pInRegMask and pArgTys arguments. This function can be called from the VS _and_ TES cases.

Reply:
Thanks,
Yes, entryArgIdx.vs.streamOutData will be deleted after tests all pass. 
I will refactor these codes in the next change after fixing this bug. :)

Reply:
Yes,  please remove vs.streamOutData. They are incorrect.